lib/LabeledTextItem/LabeledTextItem.less

```diff
@@ -10,7 +10,6 @@
 .moon-labeledtextitem .label {
 	.moon-sub-header-text;
 	color: @moon-label-text-color;
-	border-top: 3px solid @moon-label-border-color;
 	margin: 0px 0px 3px 0px;
 	padding: 6px 0px 0px 0px;
```

Comment:
I believe we can safely drop the `border-top`, `margin`, and `padding`, adding only `margin-bottom: 6px;` in their place. On `.moon-labeledtextitem` above, we can also remove the lines for `height` and `margin`.

* `.label`'s padding just pushes the title farther from the top, so it can be dropped.
* Increasing `.label`'s `margin-bottom` makes its text about the same distance from the top as it is from the value text, which more closely matches ExpandablePicker's style.
* `height: auto` isn't needed (like @ryanjduffy said in Jira) since height is already inherently "auto". One less rule to parse, yay.
* margin on `.moon-labeledtextitem` has no effect, as it's naturally got no margin. Don't need a rule to reiterate it.